test/test_warp_load.cu

```diff
@@ -28,54 +28,47 @@
 // Ensure printing of CUDA runtime errors to console
 #define CUB_STDERR
 
-#include <iterator>
-
-#include <cub/warp/warp_load.cuh>
 #include <cub/iterator/cache_modified_input_iterator.cuh>
 #include <cub/iterator/discard_output_iterator.cuh>
 #include <cub/util_allocator.cuh>
+#include <cub/warp/warp_load.cuh>
 
 #include <thrust/device_vector.h>
 #include <thrust/host_vector.h>
 #include <thrust/sequence.h>
 
+#include <iterator>
+
+#include "fill_striped.cuh"
```

Comment:
Is this the only actual difference with main in this file? We can keep formatting here, but let's postpone formatting whole files going forward. Auto-formatting existing code requires extracting the auto-formatting commit and adding it into a list that leads to ignoring that commit in the blame history. Clang format allows you to format your changes only.